Week2/week2_homework/grade_calculator.js

```diff
@@ -0,0 +1,39 @@
+"use strcit";
+
+
+// function to insert the score and calculate the grade & the percentage requested.
+function calculate_test_score(score) {
+  // 1- convert the score into a percentage
+  const full_score = 100;
+  const score_percentage = (score / full_score) * 100;
```

Comment:
What happens if `score_percentage` end up being fractional, eg `89.95`? The conditions below do not work and will result in `You got a F (89.95%)!`.

2 ways to fix this:
1. Use `Math.round` or `Math.floor` here so that it is always a whole number,
2. Change your conditional expressions below to be something like: `score_percentage >= 90 && score_percentage <= 100`, `score_percentage >= 80 && score_percentage < 90`, etc

Reply:
Thanks for the detailed feedback, I'll update it asap.